recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-core/packagegroups/packagegroup-xt-core-pv.bb

```diff
@@ -8,4 +8,5 @@ RDEPENDS_packagegroup-xt-core-pv = "\
     libxenbe \
     displbe \
     sndbe \
+    camerabe \
```

Comment:
May it should be wrapped by feature?

Reply:
Yes, next patch does that https://github.com/xen-troops/meta-xt-prod-devel/pull/246/commits/9e91aa4665c9cd1a8bab469f37bfd13aa35b847a#diff-fe7cf7d393e7b827edb506c882f7a8ddR11

Reply:
Thanks. I missed it.